spatial/curve/hilbert_test.go

```diff
@@ -342,20 +356,22 @@ func adjacent(v, u []int) bool {
 	return n == 1
 }
 
-func testCurveCase(t *testing.T, c curve, order int, want []int) {
+// testCurveCase verifies that the curve produces the expected sequence of
+// values.
+func testCurveCase(t *testing.T, c curve, order int, expected []int) {
 	t.Helper()
 
 	dim := len(c.Dims())
-	got := make([]int, len(want))
-	for i := range want {
+	actual := make([]int, len(expected))
```

Comment:
Sad. Not worth reverting, but we lean towards the English versus the Latin.

Reply:
I tend towards got/want but using "expected" made it easier to write the function comment naturally. That way I can say "produces the expected sequence of values" and it's obvious (I hope) that expected sequence corresponds to the 'expected' parameter.

Reply:
I suppose "desired sequence" and 'want' might be sufficiently clear?